manifest/com.__Choco__/SimpleScalableMeshOrbs/info.json

```diff
@@ -0,0 +1,19 @@
+{
+	"name": "SimpleScalableMeshOrbs",
+	"id": "com.__Choco__.SimpleScalableMeshOrbs",
+	"description": "Stops mesh orbs from spinning and makes them scalable.",
+	"category": "Misc",
+	"sourceLocation": "https://github.com/AwesomeTornado/SimpleScalableMeshOrbs",
+	"versions": {
+		"1.0.0": {
+			"releaseUrl": "https://github.com/AwesomeTornado/SimpleScalableMeshOrbs/releases/tag/V1.0.0",
+			"artifacts": [
+				{
+					"url": "https://github.com/AwesomeTornado/SimpleScalableMeshOrbs/releases/download/V1.0.0/SimpleScalableMeshOrbs.dll",

```

Comment:
Non:blocking: Logging the mod is loading is redundant as the mod loader already will do so `ResoniteMod.Msg("SimpleScalableMeshOrbs Loaded!");`

Additionally may be better to postfix and use the configs to remove stuff instead of copying the entire method in. It works but may require updates in the future that could be less obvious. and it becomes potentially more likely to conflict should another mod interact with this method.